src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -171,60 +172,68 @@ docker run <OTHER-DOCKER_OPTIONS> -e CHE_HOST=<ip-addr-or-dns> eclipse/che:<vers
 
 *WebSockets*
 
-Che relies on web sockets to stream content between workspaces and the browser. We have found many networks and firewalls to block portions of Web socket communications. If there are any initial configuration issues that arise, this is a likely cause of the problem.
+Che relies on WebSockets to stream content between workspaces and the browser. There are many networks and firewalls that block portions of the WebSocket communication. If there are any initial configuration issues that arise, this is a likely cause of the problem.
 
 *Topology*
 
-The Che server runs in its own Docker container, "Che Docker Container", and each workspace gets an embedded runtime which can be a set of additional Docker cotainers, "Docker Container(n)". All containers are managed by a common Docker daemon, "docker-ip", making them siblings of each other. This includes the Che server and its workspaces - each workspace runtime environment has a set of containers that is a sibling to the Che server, not a child.
+The Che server runs in its own Docker container, *Che Docker Container*. Each workspace has an embedded runtime that can be a set of additional Docker cotainers, *Docker Container(n)*. All containers are managed by a common Docker daemon, *docker-ip*, making them siblings of each other. This includes the Che server and its workspaces. Each workspace runtime environment has a set of containers that is a sibling to the Che server and is not a child.
 
 *Connectivity*
 
-The browser client initiates communication with the Che server by connecting to `che-ip`. This IP address must be accessible by your browser clients. Internally, Che runs on Tomcat which is bound to port `8080`. This port can be altered by setting `CHE_PORT` during start or in your `che.env`.
+The browser client initiates communication with the Che server by connecting to the `che-ip`. This IP address must be accessible by your browser clients. Internally, Che runs on Tomcat that is bound to the port `8080`. This port can be altered by setting the `CHE_PORT` variable during start or in the `che.env` file.
 
-When a user creates a workspace, the Che server connects to the Docker daemon at `docker-ip` and uses the daemon to launch a new set of containers that will power the workspace. These workspace containers will have a Docker-configured IP address, `workspace-container-ip`. The `workspace-container-ip` isn’t usually reachable by your browser host, `docker-ip` will be used to establish the connections between the browser and workspace containers.
+When a user creates a workspace, the Che server connects to the Docker daemon at `docker-ip` and uses the daemon to launch a new set of containers that power the workspace. These workspace containers have a Docker-configured IP address, `workspace-container-ip`. The `workspace-container-ip` is not usually reachable by your browser host. And, `docker-ip` is used to establish the connections between the browser and workspace containers.
```

Comment:
"And" is unnecessary here. The connection between both sentences is clear without it.